app/.npmrc

```
@@ -1,3 +1,3 @@
 runtime = electron
 disturl = https://electronjs.org/headers
-target = 19.0.0
+target = 21.3.3
```

I understand this is WIP, but just in case…
```suggestion
target = 22.0.0
```